core/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -940,23 +944,14 @@ protected void onException(Channel channel, Exception e) throws IOException {
         } else if (e instanceof TcpTransport.HttpOnTransportException) {
             // in case we are able to return data, serialize the exception content and sent it back to the client
             if (isOpen(channel)) {
-                final Runnable closeChannel = () -> {
+                final Consumer<Exception> closeChannel = (ex) -> {
```

Comment:
Should `e1` be suppressed to `ex` here? Also, the parentheses around `(ex)` are unnecessary.